ext/RMagick/rmmain.c

```diff
@@ -124,7 +124,7 @@ rm_aligned_malloc_size(void *ptr)
 // Refered to https://github.com/ImageMagick/ImageMagick/blob/master/MagickCore/memory-private.h
 #define MAGICKCORE_SIZEOF_VOID_P 8
 #define CACHE_LINE_SIZE  (8 * MAGICKCORE_SIZEOF_VOID_P)
-
+    size_t _aligned_msize(void *memblock, size_t alignment, size_t offset);
```

Comment:
Would it not be better to include the correct header file for this? Not sure which header file though.

Reply:
I searched for a while since I first implemented it, but couldn't find a suitable header file....

Reply:
What have you tried? Does `#include <malloc.h>` not work for this?

Reply:
Yep I tried, unfortunately, it was no effect.